src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -504,6 +504,72 @@ describe('manage nsfs cli account flow', () => {
             const res = await exec_manage_cli(type, action, account_options);
             expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.InvalidAccountName.code);
         });
+
+        it('cli account add - cli create account with supplemental groups)', async function() {
+            const { type, new_buckets_path, uid, gid, name } = defaults;
+            const supplemental_groups = '303,211';
+            const expected_groups = [303, 211];
+            const account_options = { config_root, name, new_buckets_path, uid, gid, supplemental_groups};
+            const action = ACTIONS.ADD;
+            await fs_utils.create_fresh_path(new_buckets_path);
+            await fs_utils.file_must_exist(new_buckets_path);
+            await set_path_permissions_and_owner(new_buckets_path, account_options, 0o700);
+            await exec_manage_cli(type, action, account_options);
+            const account = await config_fs.get_account_by_name(name, config_fs_account_options);
+            expect(account.nsfs_account_config.supplemental_groups).toStrictEqual(expected_groups);
+        });
+
+        it('cli account add - cli create account with supplemental groups - single group (string))', async function() {
+            const { type, new_buckets_path, uid, gid, name } = defaults;
+            const supplemental_groups = "303";
```

Comment:
Please change single quotes to double quotes.
```suggestion
            const supplemental_groups = '303';
```